samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
+be used implementing ieee802154 stack inside Linux.
+
+Requirements
+************
+
+- Linux machine host connected via USB to Zephyr WPANUSB enabled MCU
+- WPANUSB kernel driver (in the process of being open sourced)
+- IWPAN host tools at [http://wpan.cakelab.org/releases/](http://wpan.cakelab.org/releases/)
+
+Building and Running
+********************
+
+There are configuration files for various setups in the
+samples/net/wpanusb directory:
+
+- :file:`prj.conf`
+  This is the standard default config.  This can be used by itself for
```

Comment:
To have this displayed nicely, indent the "body" under these file names by two spaces:

    :file:``prj.conf``
      This is the standard default config.  This can be used by itself for
      hardware which has native 802.15.4 support.

    :file:`overlay-cc1200.conf`
      This overlay config enables support for CC1200

and so on.